docs/exceptions/exceptions-spans.md

```diff
@@ -11,33 +11,6 @@ exceptions associated with spans.
 
 <!-- toc -->
 
-- [Recording an Exception](#recording-an-exception)
```

Comment:
It's a duplicate of the spec doc https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md

Reply:
If/when this gets merged, we need to deal with the `exception.escaped` in the spec document

Reply:
absolutely, I created this draft - https://github.com/open-telemetry/opentelemetry-specification/pull/4368, PTAL